srcpkgs/pulseaudio/template

```diff
@@ -76,3 +76,19 @@ pulseaudio-devel_package() {
 		vmove usr/share/vala
 	}
 }
+
+pulseaudio-utils_package() {
+	short_desc+=" - utilities"
+	pkg_install() {
+		for b in pa-info pacat pacmd pactl padsp pamon paplay parec \
+				 parecord pasuspender pax11publish; do
+			vmove usr/bin/$b
+		done
+		for b in pacat pacmd pactl padsp pamon paplay parec \
+				 parecord pasuspender pax11publish; do
+			vmove usr/share/man/man1/${b}.1
+		done
```

Comment:
```suggestion
		for b in pa-info pacat pacmd pactl padsp pamon paplay parec \
				 parecord pasuspender pax11publish; do
			vmove usr/bin/$b
			vmove usr/share/man/man1/${b}.1
		done
```

Reply:
There is no `/usr/share/man/man1/pa-info.1`, is that OK?

Reply:
Then `vmove` it outside of the loop ;)

Reply:
> Then `vmove` it outside of the loop ;)

r/todayilearned